src/services/containerServices.ts

```diff
@@ -87,4 +87,24 @@ export class ECSContainer implements IContainerServices {
     }
     return '';
   }
+
+  async stopZombieECSTask(taskId: string) {
+    const clusterName = this._config.get<string>('taskDefinitionFamily');
+    try {
+      const res = await this._client.describeTasks({
+        cluster: clusterName,
+        tasks: [taskId],
+      });
+
+      // Only stop ECS task if it's still running
+      if (res.tasks?.[0].lastStatus === 'RUNNING') {
+        await this._client.stopTask({
+          cluster: clusterName,
+          task: taskId,
+        });
```

Comment:
This was recently updated to ensure that `stopTask` was only called if the described task instance is still running. 

BEFORE MERGING: I should double-check that this logic does what it should and the proper IAM privileges are set